blog/keycloak-newsletter-1.ftl

```diff
@@ -1,53 +1,53 @@
 <#--
 title = Keycloak Community Newsletter #1
-date = 2019-03-11
-publish = false
+date = 2019-03-26
+publish = true
 author = Sébastien Blanc
 -->
 
 <p>
-intro
+This is the very first "Keycloak Community Newsletter". The goal of this newsletter is to share the news around the Keycloak project : contributions, technical articles, future plans of the team etc...  
```

Comment:
Replace:
The goal of this newsletter is to share the news around the Keycloak project : contributions, technical articles, future plans of the team etc... 

With:

The goal of this newsletter is to share news around the Keycloak project.